Assets/SEE/UI/FilePicker/FilePickerDesktop.cs

```diff
@@ -133,7 +133,7 @@ protected override void StartDesktop()
                 FileBrowser.ShowLoadDialog(HandleFileBrowserSuccess,
                                            () => { },
                                            allowMultiSelection: false,
-                                           pickMode: FileBrowser.PickMode.Files,
+                                           pickMode: FileBrowser.PickMode.FilesAndFolders,
```

Comment:
Maybe. 

I just changed it, since some file pickers need to be able to select a folder.
(and only files were possible before)

So for now all file pickers are able to select file or folders.